gobblin-utility/src/main/java/org/apache/gobblin/util/filesystem/ThrottledFileSystem.java

```diff
@@ -53,13 +53,18 @@
    * Factory for {@link ThrottledFileSystem}.
    */
   public static class Factory<S extends ScopeType<S>> extends FileSystemInstrumentationFactory<S> {
+    private final String SERVICE_NAME_CONF_KEY = "gobblin.broker.limiter.serviceName";
```

Comment:
I think the service name is a property of the file system and not the limiter? Maybe the key should be `gobblin.broker.filesystem.limiterServiceName`?